### DISCUSSION TITLE
Github status check with manual (workflow_dispatch) actions

### DISCUSSION THREAD

**Discussion by g1soori** [community/community#32990] [Actions]
Hi Team,

I want to add a status check from one of the workflows in our repo. But this workflow is only running with workflow_dispatch trigger because it takes very long to complete and we don't want to run it with automated triggers. I was able to add the jobs of this workflow as status checks under the branch protection rules. 
Once the PR raised status check appears as "Expected — Waiting for status to be reported" but it doesn't get updated once I run the workflow manually. Does that mean workflow_dispatch workflow runs are not accepted for status checks ? I tried the same with issue_comment on PR but it doesn't work as well. Appreciate if you can advise on this.

**Comment by tnwill** [comment]
Hi @g1soori, were you able to find the solution to this? I have the same issue.

Thanks.
